examples/gemma/gemma.py

```diff
@@ -14,7 +14,7 @@
     input_tokens = tokenizer.encode(text)
 
     params=og.GeneratorParams(model)
-    params.max_length = 64
+    params.set_search_options({"max_length":256})
```

Comment:
Was it intentional change the max from 64 to 256?

Reply:
yes, want to generate longer to interactive case and short for the demon cases